selenium_tests/basic_test.py

```diff
@@ -28,3 +40,54 @@ def test_zero_price_purchase(self):
             "Something went wrong. You paid for this course but are not enrolled. Contact us for help."
         )
         self.assert_console_logs()
+
+    def test_learners(self):
+        """
+        Look at the learners page
+        """
+        self.login_via_admin(self.user)
+        self.get(self.live_server_url)
+
+        Role.objects.create(
+            user=self.user,
+            program=self.program,
+            role=Staff.ROLE_ID,
+        )
+
+        page_size = settings.ELASTICSEARCH_DEFAULT_PAGE_SIZE
+        with mute_signals(post_save):
+            for _ in range((page_size * 2) - 5):
+                profile = ProfileFactory.create(filled_out=True)
+                ProgramEnrollment.objects.create(program=self.program, user=profile.user)
+
+        other_program = ProgramFactory.create(live=True)
+        ProgramEnrollment.objects.create(
+            user=self.user,
+            program=other_program,
+        )
+
+        # Update for new users and new role
+        index_program_enrolled_users(ProgramEnrollment.objects.iterator())
+        self.get("{}/learners".format(self.live_server_url))
+        assert self.selenium.execute_script("return document.querySelectorAll('.learner-result').length") == page_size
+        self.selenium.find_elements_by_class_name('sk-pagination-option')[1].click()
+        # Verify that second page has 5 less elements
+        self.wait().until(
+            lambda driver: driver.execute_script(
+                "return document.querySelectorAll('.learner-result').length"
+            ) == page_size - 5
+        )
+
+        # Go to profile and back to learners to verify that nothing breaks
+        self.selenium.find_element_by_class_name("user-menu").click()
+        self.wait().until(
+            lambda driver: "open" in driver.find_element_by_class_name("user-menu-dropdown").get_attribute("class")
+        )
+        self.selenium.find_element_by_css_selector(
+            ".user-menu-dropdown a[href='/learner/{}']".format(self.username)
+        ).click()
```

Comment:
I think this is better:
```python
profile_link_selector = ".user-menu-dropdown a[href='/learner/{username}']".format(
    username=self.username
)
self.selenium.find_element_by_css_selector(profile_link_selector).click()
```

It would be even better if we were using Python 3.6 and could use f-strings for formatting...